app/mailers/application_mailer.rb

```diff
@@ -1,3 +1,28 @@
 class ApplicationMailer < Mail::Notify::Mailer
+  rescue_from Notifications::Client::BadRequestError, with: :handle_notify_exception
+
   GENERIC_NOTIFY_TEMPLATE = '2744ea53-34f1-431f-8173-8388fadd826a'.freeze
+
+  def handle_notify_exception(e)
+    exception = case e.message
+                when 'BadRequestError: Can’t send to this recipient using a team-only API key'
+                  NotifyTeamOnlyAPIKeyError.new(e.message)
+                else
+                  NotifyOtherBadRequestError.new
+                end
+
+    Raven.capture_exception(exception)
```

Comment:
@tijmenb When I tested locally I wasn't able to trigger Sentry without it. What do you recommend?

Reply:
@chubberlisk does setting `SENTRY_DSN` work locally?

Reply:
@tijmenb Yup!

Reply:
That means we can drop the Raven.capture_exception or not?

Reply:
We're dropping `Raven.capture_exception`. Theo and I tested in QA and it triggered twice like you said it would. Don't know why in development it only triggered once and when Raven was called.

Reply:
@chubberlisk thinking about it now: it could be because the application was not running in production mode. In development mode, errors are handled differently, e.g. you get the pretty REPL.